packages/svelte-meta-tags/src/lib/MetaTags.svelte

```diff
@@ -1,24 +1,43 @@
 <script lang="ts">
   import type { MetaTagsProps } from './types';
 
-  export let title: MetaTagsProps['title'] = '';
-  export let titleTemplate: MetaTagsProps['titleTemplate'] = '';
-  export let robots: MetaTagsProps['robots'] = 'index,follow';
-  export let additionalRobotsProps: MetaTagsProps['additionalRobotsProps'] = undefined;
-  export let description: MetaTagsProps['description'] = undefined;
-  export let mobileAlternate: MetaTagsProps['mobileAlternate'] = undefined;
-  export let languageAlternates: MetaTagsProps['languageAlternates'] = undefined;
-  export let twitter: MetaTagsProps['twitter'] = undefined;
-  export let facebook: MetaTagsProps['facebook'] = undefined;
-  export let openGraph: MetaTagsProps['openGraph'] = undefined;
-  export let canonical: MetaTagsProps['canonical'] = undefined;
-  export let keywords: MetaTagsProps['keywords'] = undefined;
-  export let additionalMetaTags: MetaTagsProps['additionalMetaTags'] = undefined;
-  export let additionalLinkTags: MetaTagsProps['additionalLinkTags'] = undefined;
-
-  $: updatedTitle = titleTemplate ? (title ? titleTemplate.replace(/%s/g, title) : title) : title;
-
-  let robotsParams = '';
+  interface Props {
+    title?: MetaTagsProps['title'];
+    titleTemplate?: MetaTagsProps['titleTemplate'];
+    robots?: MetaTagsProps['robots'];
+    additionalRobotsProps?: MetaTagsProps['additionalRobotsProps'];
+    description?: MetaTagsProps['description'];
+    mobileAlternate?: MetaTagsProps['mobileAlternate'];
+    languageAlternates?: MetaTagsProps['languageAlternates'];
+    twitter?: MetaTagsProps['twitter'];
+    facebook?: MetaTagsProps['facebook'];
+    openGraph?: MetaTagsProps['openGraph'];
+    canonical?: MetaTagsProps['canonical'];
+    keywords?: MetaTagsProps['keywords'];
+    additionalMetaTags?: MetaTagsProps['additionalMetaTags'];
+    additionalLinkTags?: MetaTagsProps['additionalLinkTags'];
+  }
+
+  let {
+    title = '',
+    titleTemplate = '',
+    robots = 'index,follow',
+    additionalRobotsProps = undefined,
+    description = undefined,
+    mobileAlternate = undefined,
+    languageAlternates = undefined,
+    twitter = undefined,
+    facebook = undefined,
+    openGraph = undefined,
+    canonical = undefined,
+    keywords = undefined,
+    additionalMetaTags = undefined,
+    additionalLinkTags = undefined
+  }: Props = $props();
```

Comment:
Remove Props interface and change partial <MetaTagsProps>?

```diff
- }: Props = $props();
+ }: Partial<MetaTagsProps> = $props();
```

Reply:
Thank you! Fixed!